ros2component/ros2component/verb/types.py

```diff
@@ -25,7 +24,6 @@ class TypesVerb(VerbExtension):
     """Output a list of components registered in the ament index."""
 
     def add_arguments(self, parser, cli_name):
-        add_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.